configuration/driver.go

```diff
@@ -0,0 +1,10 @@
+package configuration
+
+import (
+	"github.com/01org/ciao/payloads"
+)
+
+type driver interface {
+	fetchConfiguration(uri string) (payloads.Configure, error)
+	storeConfiguration(payloads.Configure) error
+}
```

Comment:
Please document fetchConfiguration() and storeConfiguration() for anyone that would eventually implement a new configuration driver to know what they're for.